src/storage/mod.rs

```diff
@@ -30,7 +30,7 @@ pub use self::types::{Key, Value, KvPair};
 pub type Callback<T> = Box<FnBox(Result<T>) + Send>;
 
 pub type CfName = &'static str;
-pub const DEFAULT_CFS: &'static [CfName] = &[];
+pub const DEFAULT_CFS: &'static [CfName] = &["lock"];
```

Comment:
RocksDB opens default cf implicitly, but I agree to add a 'default' here to make it more clear.